creation/lib/cvWCreate.py

```diff
@@ -60,7 +60,11 @@ def create_client_mapfile(mapfile_fname, my_DN, factory_DNs, schedd_DNs, collect
 
     """
     with open(mapfile_fname, "w") as fd:
-        fd.write('GSI "^{}$" {}\n'.format(re.escape(my_DN), "me"))
+        try:
+            # issue 66 transition from GSI, keep going if no DNs for proxies
+            fd.write('GSI "^{}$" {}\n'.format(re.escape(my_DN), "me"))
+        except:
```

Comment:
Try to avoid generic `except`. What is causing the problem? check if my_DN is None

Reply:
rewrote checking my_DN instead of using broad try:except: